src/helpers/actionHandler.js

```diff
@@ -6,7 +6,7 @@ export const actionHandler = (data) => {
       time: current.observation_time,
       temperature: current.temperature,
       humidity: current.humidity,
-      isDay: current.is_day,
+      isDay: "yes" ? true : false,
```

Comment:
wtf?

```suggestion
      isDay: current.is_day === "yes" ? true : false,
```

Reply:
i cant add suggestion for 1 line

export const actionHandler = ({ current }) => {